sdk/core/core-auth/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@azure/core-auth",
-  "version": "1.5.1",
+  "version": "1.6.0",
```

Comment:
@jeremymeng do you know why the ci failed?
https://dev.azure.com/azure-sdk/public/_build/results?buildId=3454462&view=logs&j=3dc8fd7e-4368-5a92-293e-d53cefc8c4b3&t=acc99708-be46-50ff-f197-4ecb2c3cab18&l=675

Reply:
reverted my last recorder PR